ETL/main.py

```diff
@@ -0,0 +1,196 @@
+import psycopg2
+import sys
+import os
+import json
+from airtable import Airtable
+import pandas
+from geopy.geocoders import Nominatim
+
+AIRTABLE_API_KEY = os.environ.get('AIRTABLE_API_KEY')
+AIRTABLE_BASE_ID = os.environ.get('AIRTABLE_BASE_ID')
+
+if AIRTABLE_API_KEY == None or AIRTABLE_API_KEY == '':
+    print('The Airtable API key is missing')
+    sys.exit(10)
+
+if AIRTABLE_BASE_ID == None or AIRTABLE_BASE_ID == '':
+    print('The Airtable Base ID is missing')
+    sys.exit(11)
+
+DATABASE_URL = os.environ.get('DATABASE_URL')
+
+if DATABASE_URL == None or DATABASE_URL == '':
+    if sys.argv[1] != None and sys.argv[1] != '':
+        DATABASE_URL = sys.argv[1]
+
+if DATABASE_URL == None or DATABASE_URL == '':
+    print('The postgres connection string DATABASE_URL is missing')
+    sys.exit(12)
+
+# Connect to the database
+db = psycopg2.connect(DATABASE_URL)
+cursor = db.cursor()
+
+# --------- Helper Functions --------- #
+
+# Log activity to a postgres table
+
+
+def log(message):
+    query = ("select etl_log('{msg}');").format(msg=message)
+    cursor.execute(query)
+    db.commit()
+
+# Create a table in postgres
+
+
+def create_import_table(table_name, json_data):
+    # Data from airtables is NOT homogenous
+    # Normalize the JSON data to simplify processing
+    data = pandas.json_normalize(json_data)
+
+    # Strip off the 'fields.' prefix that pandas adds to the column names
+    columns = data.columns
+    new_columns = []
+    for column in columns:
+        new_columns.append(str(column).replace(
```

Comment:
List comprehensions tend to be more modern / readable but gain this is just a suggestion.

Reply:
So you mean something like this: columns = [str(column).replace("fields.", "").replace(" ", "_").replace("-", "")) for column in data.columns]?